vulcanai/tests/models/test_metrics.py

```diff
@@ -294,13 +293,11 @@ def test_get_auc(self, metrics):
 
     def test_run_test(self, metrics, cnn_class):
         """Test that run_test returns values as expected."""
+        warnings.filterwarnings("ignore", category=sklearn.exceptions.UndefinedMetricWarning)
         num_items = 300
 
-        test_input = torch.Tensor(np.random.randint(0, 10,
-                                                    size=(num_items,
-                                                          *cnn_class.in_dim)))
-        test_target = torch.LongTensor(np.random.randint(0, 10,
-                                                         size=num_items))
+        test_input = torch.Tensor(np.random.randint(0, 10,size=(num_items,*cnn_class.in_dim)))
```

Comment:
the length of this line may exceed to 79 char limit which is why it was formatted as it was